integrations/dynatrace/config.yaml

```diff
@@ -0,0 +1,17 @@
+# This is an example configuration file for the integration service.
+# Please copy this file to config.yaml file in the integration folder and edit it to your needs.
+
+port:
+  clientId: "{{ from env PORT_CLIENT_ID }}" # Can be loaded via environment variable: PORT_CLIENT_ID
+  clientSecret: "{{ from env PORT_CLIENT_SECRET }}" # Can be loaded via environment variable: PORT_CLIENT_SECRET
+# The event listener to use for the integration service.
+eventListener:
+  type: KAFKA
+integration:
+  # The identifier of this integration instance.
+  identifier: "{{ from env INTEGRATION_IDENTIFIER }}"
+  # The type of the integration.
+  type: "My Integration type (Gitlab, Jira, etc.)"
+  config:
+    myGitToken: "{{ from env MY_GIT_TOKEN }}"
```

Comment:
The `config` object should match the ones you specified in the spec.yaml

you should change them to `dynatraceApiKey`, `dynatraceHostUrl`